{"name": "test_bankapp_login_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <testCases.test_user_profile_001.Test_Login01 object at 0x000001944E027750>\n\n    def test_bankapp_login_002(self):\n        # Initialize the test case\n        self.driver.get(self.login_url)\n        # Enter the username and password\n        lp = Login_Class(self.driver)\n        lp.EnterUserName(self.username)\n        lp.EnterPassword(self.password)\n        lp.ClickLoginButton()\n        time.sleep(3)\n        if self.driver.title== \"Dashboard\":\n            print(\"Test Case Passed: Login Successful\")\n            self.driver.save_screenshot(\".\\\\Screenshots\\\\test_bankapp_login_002_pass.png\")\n            assert True\n        else:\n            self.driver.save_screenshot(\".\\\\Screenshots\\\\test_bankapp_login_002_fail.png\")\n            print(\"Test Case Failed: Login Failed\")\n>           assert False\nE           assert False\n\ntestCases\\test_user_profile_001.py:63: AssertionError"}, "attachments": [{"name": "stdout", "source": "2fa6e4e3-9b7b-4778-96dc-c4a437567bbb-attachment.txt", "type": "text/plain"}], "start": 1734518261322, "stop": 1734518267591, "uuid": "01f165de-9166-4839-83c3-ae3c51b10ea4", "historyId": "96cfe49667a6d65537995dc288300f5e", "testCaseId": "96cfe49667a6d65537995dc288300f5e", "fullName": "testCases.test_user_profile_001.Test_Login01#test_bankapp_login_002", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_user_profile_001"}, {"name": "subSuite", "value": "Test_Login01"}, {"name": "host", "value": "Shree"}, {"name": "thread", "value": "15044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_user_profile_001"}]}